pdf_rendering_service/documents/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django_extensions.db.fields import CreationDateTimeField
+
+from pdf_rendering_service.documents.managers import DocumentManager
+
+
+class Document(models.Model):
+    PROCESSING, DONE = range(2)
+    STATUSES = (
+        (PROCESSING, "processing"),
+        (DONE, "done"),
+    )
```

Comment:
Have you considered using the Django (new as of 3.0) enum?
```suggestion
    class Status(models.IntegerChoices):
        PROCESSING = 0
        DONE = 1
```

https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types

Reply:
Nice, thanks for pointing me